ansible/roles/shared_dir.yml

```diff
@@ -0,0 +1,17 @@
+---
+
+- name: SharedDirHost
+  hosts: shared_dir_host  # Should be one
+  sudo: yes
+  vars_files:
```

Comment:
Not sure I see how this shares variables between roles. Seems like this vars file is only in this role. Are you just saying this is a way to do it in the future if needed?